tests/suites/test_suite_pkcs1_v15.function

```diff
@@ -14,23 +14,22 @@ void pkcs1_rsaes_v15_encrypt( int mod, int radix_N, char *input_N, int radix_E,
                                char *message_hex_string, char *seed,
                                char *result_hex_str, int result )
 {
-    unsigned char message_str[1000];
-    unsigned char output[1000];
-    unsigned char output_str[1000];
-    unsigned char rnd_buf[1000];
+    unsigned char message_str[118];
+    unsigned char output[128];
+    unsigned char output_str[257];
+    unsigned char rnd_buf[109];
```

Comment:
These were determined experimentally?

Reply:
yes, I ran the test, and printed the lengths. I put here the maximal lengths used.
This is prone to errors obviously, once new test vectors will arrive